src/modules/compressionType.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+
+const zlib = require('zlib');
+
+const getCompressionType = (compressionType) => {
+  switch (compressionType) {
+    case '.br':
+      return zlib.createBrotliCompress;
+
+    case '.dfl':
+      return zlib.createDeflate();
```

Comment:
createDeflate is a function and should be returned as such, similar to createBrotliCompress and createGzip. Remove the parentheses from createDeflate.